# 电负性

Electronegativity

## 前言

### 定义

电负性定义为分子中原子吸引电子的能力。一个原子的电负性越大，其吸引成键电子的能力就越强。

电负性概念由美国化学家莱纳斯·鲍林（Linus Pauling）于1932年首次提出。

### 问题

电负性概念提出的直接原因是为了解释——为什么异核分子的化学键键能的预测值总是低于实验值。以异核分子 $\ce{HF}$ 为例，当时人们试图通过 $\ce{H2, F2}$ 的键能来预测 $\ce{HF}$ 的键能。假设 $\ce{HF}$ 的键能是 $\ce{H2, F2}$ 的键能的算数平均值或几何平均值，于是：

$\ce{H2}$ 的键能为 432 kJ/mol，$\ce{F2}$ 的键能为 154 kJ/mol。

$\ce{HF}$ 的键能预测值为①算数平均值：(432 + 154)/2 = 293 kJ/mol，②几何平均值：√(432 * 154) = 258 kJ/mol。

$\ce{HF}$ 的键能实验值为 565 kJ/mol。

> 键能数据来源：[🔗 Bond Energies](https://chem.libretexts.org/Bookshelves/Physical_and_Theoretical_Chemistry_Textbook_Maps/Supplemental_Modules_(Physical_and_Theoretical_Chemistry)/Chemical_Bonding/Fundamentals_of_Chemical_Bonding/Bond_Energies)

$\ce{HF}$ 的键能预测值与实验值的差异表明，$\ce{HF}$ 的键能不能简单地通过 $\ce{H2, F2}$ 的键能的算数平均值或几何平均值来预测。其它许多实验也证明了这一点（但是我不想复刻计算^_^）。

### 解释

为了解释这一现象，鲍林提出了电负性的概念。他认为，异核分子的原子之间除了共价电子吸引力，还存在一种额外的**静电引力**。

在新的模型下，原子之间并非平等地分享电子对，而是电子云向电负性更大的原子偏移。这种偏移导致电荷分布不均：一端原子积累了部分负电荷（$\delta-$），另一端则显现出部分正电荷（$\delta+$）。

这种因正负电性产生的额外库仑力，增强了化学键的稳定性，从而在宏观上表现为键能实验值高于预测值。原子之间的电负性之差越大，这种电荷偏移就越剧烈，化学键的离子性也就越强。

另外，这不仅增强了键能，还缩短了键长（涉及到“极性收缩”）。

### 数学表达

原子之间的电负性之差，正比于键能实验值与键能预测值之差（额外能量 $\Delta$）的平方根。

假设原子 $A$ 和 $B$ 的电负性分别为 $\chi_A$ 和 $\chi_B$，则有：

$$
\left| \chi_A - \chi_B \right| = k \cdot \sqrt{\Delta}
$$

其中：

- $\Delta = E_{exp} - E_{cov}$
- $E_{cov} = \sqrt{E(A-A) \cdot E(B-B)}$

## 鲍林尺度

Pauling Scale

### 定义

现代化学通过以下约定确立了鲍林电负性的定量标准：

- 原子基准：规定氢原子的电负性为 $\chi_{\ce{H}} = 2.20$。作为周期表中分布最广的元素，它是整个标度的“锚点”。
- 比例系数：规定系数 $k = 0.102 \text{ (kJ/mol)}^{-1/2}$。\
注：该常数旨在将以 $\text{kJ/mol}$ 为单位的键能偏差 $\Delta$ 映射为无量纲的电负性数值。




In [13]:
import pint
ureg = pint.UnitRegistry()
Q_ = ureg.Quantity

k = Q_(1, 'eV/particle')
print(f"k = {k.to('kJ/mol')**(-1/2):.3g}")

k = 0.102 mole ** 0.5 / kilojoule ** 0.5


- 计算公式：任意原子对的电负性差定义为：
$$
\left| \chi_A - \chi_B \right| = 0.102 \sqrt{\Delta / (\text{kJ/mol})}
$$

各元素原子的电负性，参考数据：[🔗 Electronegativity](https://en.wikipedia.org/wiki/Electronegativity)

### 验证

我们之前通过[计算](#问题)发现，$\ce{HF}$ 的键能预测值（几何平均值） $258 \text{ kJ/mol}$ 低于实验值 $565 \text{ kJ/mol}$。

通过查表，可知 $\chi_{\ce{H}} = 2.20$，$\chi_{\ce{F}} = 3.98$，$k = 0.102$，$E(H-H) = 432 \text{ kJ/mol}$，$E(F-F) = 154 \text{ kJ/mol}$。于是有：

In [18]:
#| code-fold: false

import numpy as np

X_H = 2.20
X_F = 3.98
k = 0.102
E_HH = 432
E_FF = 154

E_HF = np.sqrt(E_HH * E_FF) + (np.abs(X_H - X_F) / k)**2
print(f"E_HF = {E_HF:.3g}")

E_HF = 562


使用鲍林电负性模型，可以计算出 $\ce{HF}$ 的键能预测值为 $562 \text{ kJ/mol}$，与实验值 $565 \text{ kJ/mol}$ 非常接近。这证明了电负性模型的可行性。

### 化学键连续性

电负性概念的提出，不仅解释了异核分子的键能预测值低于实验值的现象，还揭示了化学键的连续性。它将共价键与离子键统一在同一个数学框架下。化学键不再是孤立的分类，而是一个关于电负性差异的连续函数。

| 物质 | 电负性差 (Δχ) | 键类型 (定性) |
| :---: | :---: | :---: |
| $\ce{F2}$ | 3.98 - 3.98 = 0.00 | 非极性共价键 |
| $\ce{CH4}$ | 2.55 - 2.20 = 0.35 | 非极性共价键 (弱极性) |
| $\ce{HCl}$ | 3.16 - 2.20 = 0.96 | 极性共价键 |
| $\ce{HF}$ | 3.98 - 2.20 = 1.78 | 极性共价键 (强极性) |
| $\ce{NaCl}$ | 3.16 - 0.93 = 2.23 | 离子键 |
| $\ce{CsF}$ | 3.98 - 0.79 = 3.19 | 离子键 (极强离子性) |

从 $\ce{F2}$ 到 $\ce{CsF}$，电子云经历了从“完美对称共享”到“几乎完全剥离”的过程。通过鲍林尺度，我们证明了离子键本质上只是极性极强的共价键。这种从量变到质变的视角，正是现代结构化学的基石。